src/components/casestudyandcoretools/Casestudy.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Entries, Entry } from "@/types/interfaces";
+import InformationCard from "../cards/InformationCard";
+import { ArcherContainer } from "react-archer";
+
+interface Props {
+  categorizedEntries: { level1: Entries; level2: Entries; level3: Entries };
+}
+
+const Casestudy = ({ categorizedEntries }: Props) => {
```

Comment:
Can you change this into camecase like CaseStudy as well as its file name

Reply:
Removed